azure/create-platform-prerequisites/variables.tf

```diff
@@ -40,22 +38,24 @@ variable "stage" {
       "QA",
       "IA",
       "EA",
+      "Doc",
+      "Support",
       "Demo",
-      "Prod"
-    ], var.stage)
-    error_message = "Stage must be either: OnBoarding, Dev, QA, IA, EA, Demo, Prod."
+      "Prod",
+      "Uat"
```

Comment:
what does 'uat' mean ?

Reply:
User Acceptance but i'll update the name to preprod